.editorconfig

```
@@ -55,6 +55,11 @@ trim_trailing_whitespace = unset
 [*.lock]
 indent_size = unset
 
+# trailing whitespace is an actual syntax element of classic Markdown/
+# CommonMark to enforce a line break
+[**.md]
```

We don't need to match path separators at the top level so we only need one `*`, consistent with the others above it.

```suggestion
[*.md]
```